src/sentry/static/sentry/app/components/hoverCard.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import Avatar from './avatar';
+
+import LoadingIndicator from './loadingIndicator';
+import LoadingError from './loadingError';
+
+import ApiMixin from '../mixins/apiMixin';
+
+const HoverCard = React.createClass({
```

Comment:
This is not just any hover card. This is a `VersionHoverCard`. Please rename the class and file.